FirebaseAuth/Sources/Swift/ActionCode/ActionCodeSettings.swift

```diff
@@ -42,6 +42,9 @@ import Foundation
   /// The Firebase Dynamic Link domain used for out of band code flow.
   @objc open var dynamicLinkDomain: String?
```

Comment:
When I mark this as deprecated, our test failed due to build warnings.  Any suggestions?  I still want to keep the dynamicLinkDomain tests because both solutions will be available until FDL sunset.

Reply:
You can temporarily add `--allow-warnings` with a comment to the `pod lib lint` test invocations